packages/ruby/README.md

```diff
@@ -1,111 +1,30 @@
-
-### Payload Data
-
-Option              | Required? | Type             | Description
---------------------|-----------|------------------|----------
-`api_key`           | yes       | string           | API Key used to make the request. Note that this is different from the `readmeAPIKey` described above in the options data. This should be a value from your API that is unique to each of your users.
-`label`             | no        | string           | This will be the user's display name in the API Metrics Dashboard, since it's much easier to remember a name than an API key.
-`email`             | no        | string           | Email of the user that is making the call.
-`log_id`            | no        | string           | A UUIDv4 identifier. If not provided this will be automatically generated for you. Providing your own `log_id` is useful if you want to know the URL of the log in advance, i.e. `{your_base_url}/logs/{your_log_id}`.
-`ignore`            | no        | bool           | A flag that when set to `true` will suppress sending the log.
-
-### Rails
-
-```ruby
-# config/environments/development.rb or config/environments/production.rb
-require "readme/metrics"
-
-sdk_options = {
-  api_key: "<<apiKey>>",
-  development: false,
-  reject_params: ["not_included", "dont_send"],
-  buffer_length: 5,
-}
-
-config.middleware.use Readme::Metrics, sdk_options do |env|
-  current_user = env['warden'].authenticate
-
-  payload_data = current_user.present? ? {
-    api_key: current_user.api_key, # Not the same as the ReadMe API Key
-    label: current_user.name,
-    email: current_user.email
-  } : {
-    api_key: "guest",
-    label: "Guest User",
-    email: "guest@example.com"
-  }
-
-  payload_data
-end
+```bash
+gem "readme-metrics"
 ```
 
-### Rack
-
-```ruby
-# config.ru
-sdk_options = {
-  api_key: "<<apiKey>>",
-  development: false,
-  reject_params: ["not_included", "dont_send"]
-}
-
-use Readme::Metrics, sdk_options do |env|
-    {
-      api_key: "owlbert_api_key"
-      label: "Owlbert",
-      email: "owlbert@example.com",
-      log_id: SecureRandom.uuid
-    }
-end
-
-run YourApp.new
-```
-
-### Sample Applications
-
-- [Rails](https://github.com/readmeio/metrics-sdk-rails-sample)
-- [Rack](https://github.com/readmeio/metrics-sdk-racks-sample)
-- [Sinatra](https://github.com/readmeio/metrics-sdk-sinatra-example)
-
-### Contributing
-
-Ensure you are running the version of ruby specified in the `Gemfile.lock`; use `rvm` to easy manage ruby versions. Run `bundle` to install dependencies, `rake` or `rspec` to ensure tests pass, and `bundle exec standardrb` to lint the code.
-
-## License
+* [Integration Documentation](https://docs.readme.com/docs/ruby-api-metrics-set-up)
```

Comment:
Same issue as the PHP one. Open to ideas to present this a little nicer.

Reply:
per above:
```suggestion
**For more information on setup, check out our [integration documentation](https://docs.readme.com/docs/ruby-api-metrics-set-up).**
```